cider/tests/utils/DuckDbQueryRunnerTest.cpp

```diff
@@ -136,7 +412,7 @@ TEST(DuckDBQueryRunnerTest, insertCiderBatchTest) {
   auto res = runner.runSql("select * from table_test;");
   CHECK(!res->HasError());
   CHECK_EQ(res->ColumnCount(), 2);
-  std::cout << "error: " << res->error << std::endl;
+  std::cout << "error: " << res->error << std::endl;  // YBRua: wrap this into an if?
```

Comment:
@jikunshang should we replace all std::cout using the `CiderException`?

Reply:
+1.

Reply:
this line has been removed